schema.xml

```diff
@@ -0,0 +1,1220 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
This shouldn't be committed.  I feel like some deja vu, I believe I called this file out on a previous review recently.

Reply:
Please work with Martin/Alva on this. I agree.